mozetl/clientsdaily/rollup.py

```diff
@@ -10,13 +10,25 @@
 )
 from fields import MAIN_SUMMARY_FIELD_AGGREGATORS
 
-ACTIVITY_SUBMISSION_LAG = DT.timedelta(10)
+LAG = 10
+ACTIVITY_SUBMISSION_LAG = DT.timedelta(LAG)
+ACTIVITY_DATE_COLUMN = F.expr(
+    "substr(subsession_start_date, 1, {})".format(LAG)
```

Comment:
This shouldn't be using `LAG` - it's just chopping off the date prefix of the date string, right?

Reply:
Doh! How silly of me.

Reply:
Easy mistake to make when extracting a constant (which is a good thing to do).